tools/functional-tester/etcd-runner/command/global.go

```diff
@@ -95,7 +108,11 @@ func doRounds(rcs []roundClient, rounds int) {
 	}
 
 	start := time.Now()
-	for i := 1; i < len(rcs)*rounds+1; i++ {
+	totalRounds := len(rcs)*rounds + 1
```

Comment:
Don't call this `totalRounds`. That value is already `rounds`. Call it `totalResults` since it's counting the total results for all rounds.